packages/bsb/docs/cli/options.rst

```diff
@@ -131,6 +131,16 @@ List of options
 
   * *env*: ``BSB_CONFIG_FILE``
 
+* ``simulation_report``: Add a listener to do reporting of simulation progress, the report will be updated every time step.
```

Comment:
It's tricky to find a good name that both says what it does and isn't 2 meters long, but what about `sim_console_progress` or `sim_progress_report`, `sim_to_console` :s